bin/tx-prover/README.md

```diff
@@ -61,27 +61,17 @@ max_retries_per_request = 1
 max_req_per_sec = 5
 # Interval to check the health of the workers
 health_check_interval_secs = 1
-
-[[workers]]
-host = "0.0.0.0"
-port = 8083
-
-[[workers]]
-host = "0.0.0.0"
-port = 8084
 ```
 
-To add more workers, you will need to add more items with the `[[workers]]` tags.
-
 Then, to start the proxy service, you will need to run:
 
 ```bash
-miden-tx-prover start-proxy
+miden-tx-prover start-proxy [worker1] [worker2] ... [workerN]
 ```
 
-This command will start the proxy using the workers defined in the configuration file to send transaction witness to prove.
+This command will start the proxy using the workers passed as arguments. The workers should be in the format `host:port`. If no workers are passed, the proxy will start without any workers and will not be able to handle any requests.
 
-At the moment, when a worker added to the proxy stops working and can not connect to it for a request, the connection is marked as retriable meaning that the proxy will try reaching the following worker in a round-robin fashion. The amount of retries is configurable changing the `max_retries_per_request` value in the configuration file.
+At the moment, when a worker added to the proxy stops working and can not connect to it for a request, the connection is marked as retriable meaning that the proxy will try reaching another worker. The amount of retries is configurable changing the `max_retries_per_request` value in the configuration file.
```

Comment:
typo: "The amount of retries" should be "The number of retries". Also, I'd change this sentence to be:

"The number of retries is configurable via the `max_retries_per_request` value in the configuration file."